### PR TITLE
Ability to remove bookmarks

### PR DESCRIPTION
**Pull request by ihnatz**

### CODE REVIEW

File: static/templates/bookmarks.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     </span>
     <span class="bookmarkFooter">
         <img class="bookmarkFavicon" src="{{favicon}}" />
-        <a href="{{url}}" class="bookmarkUrl">{{url}}</a>
+        <a href="{{url}}" data-id="{{id}}" class="bookmarkUrl">{{url}}</a>
```
Comment (YurySolovyov) [inline static/templates/bookmarks.html]: Let's set it on the root of bookmark, so you don't have to use `.find` twice as on line 132-133

File: src/frontend/main.js [hunk 2]
```diff
@@ -125,6 +128,16 @@ $(function() {
         }).then(clearResults);
     };
 
+    const removeBookmark = function() {
+        const bookmark = results.find('.selected');
+        const id = bookmark.find('.bookmarkUrl').data('id');
+        chrome.bookmarks.remove(String(id), function() {
```
Comment (YurySolovyov) [inline src/frontend/main.js]: Should be a string already

File: src/frontend/main.js [hunk 3]
```diff
@@ -125,6 +128,16 @@ $(function() {
...
+            selectNext();
```
Comment (YurySolovyov) [inline src/frontend/main.js:135]: What about removing last item? In that case next will be empty.

Comment (ihnatz) [inline src/frontend/main.js:135]: nothing happens

**Pull request merged**